test/ably/utils.py

```diff
@@ -62,7 +62,8 @@ def test_decorated(self, *args, **kwargs):
             for response in responses:
                 if protocol == 'json':
                     self.assertEquals(response.headers['content-type'], 'application/json')
-                    json.loads(response.text)
+                    if response.content:
```

Comment:
Not a super expert in Python, but why not testing for `response.text`, which is what you access in the subsequent line?

Reply:
Yes, I copied ``if response.content:`` from the msgpack case, just a few lines below.

``content`` is the response body as a byte string, ``text`` is the same as a text string (unicode)

I have done a different fix, and used ``response.json()`` instead of ``json.loads(...)``.